projects/rocprofiler-compute/CMakeLists.txt

```diff
@@ -615,19 +645,22 @@ add_custom_target(
     # bombs as explained in
     # https://nuitka.net/user-documentation/common-issue-solutions.html#fork-bombs-self-execution
     COMMAND
-        ${Python3_EXECUTABLE} -m nuitka --mode=onefile --no-deployment-flag=self-execution
-        --include-data-files=${PROJECT_SOURCE_DIR}/VERSION*=./ --enable-plugin=no-qt
-        --include-package=dash_svg --include-package-data=dash_svg
+        ${Python3_EXECUTABLE} -m nuitka --mode=onefile
+        --no-deployment-flag=self-execution
+        --include-data-files=${PROJECT_SOURCE_DIR}/VERSION*=./
+        --enable-plugin=no-qt --include-package=dash_svg
+        --include-package-data=dash_svg
         --include-package=dash_bootstrap_components
-        --include-package-data=dash_bootstrap_components --include-package=plotly
-        --include-package-data=plotly --include-package=kaleido
-        --include-package-data=kaleido --include-package=rocprof_compute_analyze
+        --include-package-data=dash_bootstrap_components
+        --include-package=plotly --include-package-data=plotly
+        --include-package=kaleido --include-package-data=kaleido
+        --include-package=rocprof_compute_analyze
         --include-package-data=rocprof_compute_analyze
-        --include-package=rocprof_compute_profile
-        --include-package-data=rocprof_compute_profile
-        --include-package=rocprof_compute_tui --include-package-data=rocprof_compute_tui
-        --include-package=rocprof_compute_soc --include-package-data=rocprof_compute_soc
-        --include-package=utils --include-package-data=utils rocprof-compute
+        --include-package=rocprof_compute_tui
```

Comment:
@vedithal-amd Could you please help to double check the including's here are good?